### PR TITLE
[Tests-Only] Interactions for share Recipient test added into shareRecipient file

### PR DESCRIPTION
**Pull request by jasson99**
## Related Issue:
- #500

### CODE REVIEW

**Review by swoichha — COMMENTED** [review]
Why is there only one interaction added in `shareRecipientTest` file? Is it WIP or we won't be needing any interaction for remaining three tests?

File: tests/shareRecipientTest.js [hunk 1]
```diff
@@ -93,7 +69,33 @@ fdescribe('Main: Currently testing share recipient,', function () {
...
+        headers: {
+          'Content-Type': 'application/json; charset=utf-8',
+          'Access-Control-Allow-Origin': origin
+        },
+        body: '{"ocs"' +
+          ':{"meta":{"status":"ok","statuscode":200,' +
+          '"message":"OK","totalitems":"","itemsperpage":""},' +
+          '"data":{"exact":{"users":[],"groups":[],"remotes":[]},' +
+          '"users":[{"label":"test123","value":{"shareType":0,"shareWith":"test123"}}],' +
+          '"groups":[{"label":"testGroup","value":{"shareType":1,"shareWith":"testGroup"}}],' +
```
Comment (individual-it) [inline tests/shareRecipientTest.js:95]: can we make the username and groupname dynamic

File: tests/shareRecipientTest.js [hunk 2]
```diff
@@ -19,7 +19,6 @@ fdescribe('Main: Currently testing share recipient,', function () {
   })
 
   // TESTING CONFIGS
-  const testUserPassword = config.testUserPassword
   const testUser = config.testUser
   const testGroup = config.testGroup
```
Comment (individual-it) [inline tests/shareRecipientTest.js:23]: why redefining the variables

**Review by individual-it — CHANGES_REQUESTED** [review]

File: tests/shareRecipientTest.js (hunk 1 above)
Comment (jasson99) [reply tests/shareRecipientTest.js]: fixed

File: tests/shareRecipientTest.js (hunk 2 above)
Comment (jasson99) [reply tests/shareRecipientTest.js]: fixed

**Review by individual-it — APPROVED** [review]

**Pull request merged**